### PR TITLE
Add Lingo to Installation Section

### PR DESCRIPTION
**Pull request by tanneberger**

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -150,3 +150,15 @@ Regular and nightly release builds of the command line tools can be downloaded f
...
+
+## Lingo
+
+Lingo is the package manager and build tool of the lingua franca ecosystem. It is currently under 
+development and is not yet fully stabilized. The latest information can be found in the 
+[Lingo Github Repository](https://github.com/lf-lang/lingo).
+
+```
+    cargo install lingua-franca
+    PATH=$PATH:~/.cargo/bin
```
Comment (lhstrh) [inline docs/installation.md:163]:
```suggestion
```

File: docs/installation.md [hunk 2]
```diff
@@ -150,3 +150,15 @@ Regular and nightly release builds of the command line tools can be downloaded f
...
 
 If you would like to build from source or contribute to Lingua Franca, you can find further instructions in the [developer section](./developer/downloading-and-building.mdx).
+
+
+## Lingo
+
+Lingo is the package manager and build tool of the lingua franca ecosystem. It is currently under 
+development and is not yet fully stabilized. The latest information can be found in the 
+[Lingo Github Repository](https://github.com/lf-lang/lingo).
+
```
Comment (lhstrh) [inline docs/installation.md:160]:
```suggestion

Assuming that `~/.cargo/bin` is on your `PATH`, you can install `lingo` using the following command:
```

**Review by lhstrh — APPROVED** [review]
Left a suggestion change.

File: docs/installation.md [hunk 3]
```diff
@@ -150,3 +150,15 @@ Regular and nightly release builds of the command line tools can be downloaded f
 ### From Source
 
 If you would like to build from source or contribute to Lingua Franca, you can find further instructions in the [developer section](./developer/downloading-and-building.mdx).
+
+
+## Lingo
+
+Lingo is the package manager and build tool of the lingua franca ecosystem. It is currently under
```
Comment (cmnrd) [inline docs/installation.md:157]:
```suggestion
Lingo is the package manager and build tool of the Lingua Franca ecosystem. It is currently under 
```

File: docs/installation.md (hunk 1 above)
Comment (tanneberger) [reply docs/installation.md]: done

File: docs/installation.md (hunk 2 above)
Comment (tanneberger) [reply docs/installation.md:160]: added  - done

File: docs/installation.md (hunk 3 above)
Comment (tanneberger) [reply docs/installation.md]: fixed

**Pull request merged**